website/src/components/CallToAction.tsx

```diff
@@ -1,17 +1,31 @@
-import { useColorMode } from "@chakra-ui/react";
+import { Box, Link, Text, useColorMode } from "@chakra-ui/react";
+import Image from "next/image";
+import { useTranslation } from "next-i18next";
 import { useId } from "react";
 
+import discord from "./assets/discord.svg";
```

Comment:
If we're updating this, I'd rather we use an icon from `react-icons`.  Can you use one of those and get rid of these custom local svgs?

Reply:
Thank you for the suggestion I have updated the PR to use `react-icons` and have deleted the custom `svg` assets.